dandischema/datacite.py

```diff
@@ -190,4 +196,25 @@ def to_datacite(meta: ty.Union[dict, PublishedDandiset]) -> dict:
         attributes["subjects"] = [{"subject": el} for el in meta.keywords]
 
     datacite_dict = {"data": {"id": meta.doi, "type": "dois", "attributes": attributes}}
+
+    if validate:
+        validate_datacite(datacite_dict)
+
     return datacite_dict
+
+
+def _get_datacite_schema():
```

Comment:
I know that it is just a code move.

But this better be cached or (even without any analysis of how often this is called) we might end up with all kinds fails upon some intermittent connection issues etc.
Or might be better to just include that file right here in the dandischema sources/distribution -- who knows what RFing they would decide to do in datacite etc.  Then this file would always be with us and reliably available -- would make it more robust.

Reply:
that's why i have left that by default as `False`. we should have some trust no?

people can retry on http errors for anyone using the code. i will at least make it a permalink.